src/main/java/net/minecraftforge/common/ToolActions.java

```diff
@@ -114,6 +114,8 @@
      */
     public static final ToolAction SHIELD_BLOCK = ToolAction.get("shield_block");
 
+    public static final ToolAction FISHING_ROD_CAST = ToolAction.get("fishing_rod_cast");
```

Comment:
This should probably have a one sentence javadoc like the other tool actions do.